README.md

```diff
@@ -14,30 +13,28 @@ You should consider using LXD if you want to containerize different environments
 
 ## Get started
 
-To get a better idea of what LXD is and what it does, you can [try it online](https://linuxcontainers.org/lxd/try-it/)!
+See [Getting started](https://documentation.ubuntu.com/lxd/en/stable-5.0/getting_started/) in the LXD documentation for installation instructions and first steps.
 This interactive demo guides you through the basic steps of using LXD and shows how to launch and inspect your first instances, update the configuration, work with commands and files inside an instance and create snapshots.
 
 Then, if you want to run LXD locally, take a look at our [getting started guide](https://linuxcontainers.org/lxd/getting-started-cli/) for installation instructions.
 
-- Release announcements: [`https://linuxcontainers.org/lxd/news/`](https://linuxcontainers.org/lxd/news/)
-- Release tarballs: [`https://linuxcontainers.org/lxd/downloads/`](https://linuxcontainers.org/lxd/downloads/)
-- Documentation: [`https://linuxcontainers.org/lxd/docs/stable-5.0/`](https://linuxcontainers.org/lxd/docs/stable-5.0/)
+- Release announcements: [`https://discourse.ubuntu.com/c/lxd/news/`](https://discourse.ubuntu.com/c/lxd/news/)
+- Release tarballs: [`https://github.com/canonical/lxd/releases/`](https://github.com/canonical/lxd/releases/)
+- Documentation: [`https://documentation.ubuntu.com/lxd/en/stable-5.0/`](https://documentation.ubuntu.com/lxd/en/stable-5.0/)
 
 <!-- Include end LXD intro -->
 
 ## Status
 
 Type                | Service               | Status
 ---                 | ---                   | ---
-CI (client)         | GitHub                | [![Build Status](https://github.com/lxc/lxd/workflows/Client%20build%20and%20unit%20tests/badge.svg)](https://github.com/lxc/lxd/actions)
+CI (client)         | GitHub                | [![Build Status](https://github.com/canonical/lxd/workflows/Client%20build%20and%20unit%20tests/badge.svg)](https://github.com/canonical/lxd/actions)
```

Comment:
Can we specify a particular target for the badge?

Reply:
Found it for GitHub, yes. Not sure if it makes sense for any of the other links?

Reply:
Cool lets do it for github at least then, thanks